apps/spruce/src/pages/version/VersionTabs.tsx

```diff
@@ -123,7 +127,12 @@ const tabMap = ({
       key="test-analysis-tab"
       data-cy="test-analysis-tab"
       id="test-analysis-tab"
-      name="Test Analysis"
+      name={
+        <span ref={refEl as React.Ref<HTMLDivElement>}>
+          Test Analysis
+          <Badge variant="green">New</Badge>
```

Comment:
Could you add some spacing here? Could also optionally make the badge dependent on `SEEN_TEST_ANALYSIS_TAB_GUIDE_CUE`
<img width="143" alt="image" src="https://github.com/user-attachments/assets/2498d6e9-f9e5-4756-84f3-4e767bdd7a76">

Reply:
Good idea!